sdk/external/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# We do not install printf stuff anyway, so this just removes a warning
+if (NOT DEFINED CMAKE_INSTALL_LIBDIR)
+  set(CMAKE_INSTALL_LIBDIR "")
+endif()
+
+set(BUILD_STATIC_LIBRARY True)
+
+# Configure library to be as small as possible
+set(SUPPORT_DECIMAL_SPECIFIERS false)
+set(SUPPORT_EXPONENTIAL_SPECIFIERS false)
+set(SUPPORT_MSVC_STYLE_INTEGER_SPECIFIERS false)
+set(SUPPORT_WRITEBACK_SPECIFIER false)
+set(SUPPORT_LONG_LONG false)
+set(CHECK_FOR_NUL_IN_FORMAT_SPECIFIER false)
+
+add_subdirectory(printf EXCLUDE_FROM_ALL)
```

Comment:
Why the EXCLUDE_FROM_ALL here?

Reply:
I added comments in the code, but essentially to avoid LF compiler's `make install` command from running `printf`'s install commands too. It also makes sense because `printf` is its own standalone project.